src/applications/personalization/profile-2/helpers.js

```diff
@@ -0,0 +1,47 @@
+import moment from 'moment';
+
+const asyncReturn = (returnValue, delay = 300) =>
+  new Promise(resolve => {
+    setTimeout(() => {
+      resolve(returnValue);
+    }, delay);
+  });
+
+export const asyncFetchMilitaryInformation = () =>
```

Comment:
this endpoint works locally for me. and if it is mocked locally like this I think it needs just return `serviceHistory` array, without the wrapping `militaryInformation.serviceHistory`